scripts/nix-profile-daemon.sh.in

```diff
@@ -21,18 +15,6 @@ if test -w $HOME; then
       fi
   fi
 
-  # Subscribe the root user to the NixOS channel by default.
-  if [ "$USER" = root -a ! -e $HOME/.nix-channels ]; then
```

Comment:
This may be worth keeping. I'm not sure the original use case, but it doesn't seem less relevant now.

Reply:
It's already done by the installer script, so we don't need to check for it here.